### PR TITLE
LPS-57634

### PR DESCRIPTION
**Pull request by bejondshao**

### CODE REVIEW

**Comment by bejondshao** [comment]
Hi Sam, as this pull request is committed when hugo and dale left for EVP. So I still send it to you.

Thanks
Bejond

**Comment by bejondshao** [comment]
ci:retest

**Comment by bejondshao** [comment]
ci:retest

**Comment by samuelkong** [comment]
ci:retest

**Comment by samuelkong** [comment]
ci:retest

File: modules/util/portal-tools-service-builder/src/com/liferay/portal/tools/service/builder/dependencies/persistence_impl_finder_find.ftl [hunk 1]
```diff
@@ -1179,6 +1157,84 @@ that may or may not be enforced with a unique index at the database level. Case
...
+							for (${finderCol.type} ${finderCol.name} : ${finderCol.names}) {
+								if (!InlineSQLHelperUtil.isEnabled(${finderCol.name})) {
+									notEnabledGroupIdsList.add(${finderCol.name});
+								}
+								else {
+									enabledGroupIdsList.add(${finderCol.name});
+								}
+							}
+
+							List<${entity.name}> list = new ArrayList<${entity.name}>();
```
Comment (samuelkong) [inline modules/util/portal-tools-service-builder/src/com/liferay/portal/tools/service/builder/dependencies/persistence_impl_finder_find.ftl]: As far as variable naming, `list` is not going to get approved by Brian. Maybe something like `${entity.name}s`

Comment (bejondshao) [inline modules/util/portal-tools-service-builder/src/com/liferay/portal/tools/service/builder/dependencies/persistence_impl_finder_find.ftl]: Hi Sam,

Do you mean "List<>" is not good or "list" is not good? Maybe should I use
xxxList?

Bejond Shao

Jr. Engineer, Technical Support

Subscription Services

Liferay Dalian Software Co., Ltd.

**Comment by samuelkong** [comment]
`list` is not good. You can't change `List<>`

Yes, I think "xxxList" is ok.

**Comment by samuelkong** [comment]
ci:retest

**Comment by bejondshao** [comment]
Hi Sam,

It seems my master against with your master. I'll rebase my master and
resend it to you. Thanks

Bejond Shao

Jr. Engineer, Technical Support

Subscription Services

Liferay Dalian Software Co., Ltd.

**Pull request closed without merging**